src/components/Dashboard/TodoList.vue

```diff
@@ -500,83 +500,64 @@ export default {
           filingId: filing.header.filingId,
           filingDate: filing.correction.correctedFilingDate,
           corrFilingId: filing.correction.correctedFilingId,
-          correctedFilingType: this.formatFilingType(filing.correction.correctedFilingType),
-          title: `Priority Correction - ${this.formatFilingType(filing.correction.correctedFilingType)}`,
-          draftTitle: `Correction Filing`,
+          correctedFilingType: this.typeToTitle(filing.correction.correctedFilingType),
+          title: `Priority Correction - ${this.typeToTitle(filing.correction.correctedFilingType)}`,
```

Comment:
Note that there are also regular (non-priority) corrections. The "Priority" keyword should be used only if the Priority flag was set in the staff payment options (filing.header.priority).

Reply:
K, thanks for the heads up. We can update this once that property is coming in from the back end 👍

Reply:
This property already comes from the BE ... but only if it's True.

Reply:
ahh i see. Thanks for clarifying!